pkg/sentry/devices/nvproxy/frontend.go

```diff
@@ -564,6 +564,18 @@ func ctrlHasFrontendFD[Params any, PtrParams hasFrontendFDPtr[Params]](fi *front
 	return n, nil
 }
 
+func ctrlAttachGpu(fi *frontendIoctlState, ioctlParams *nvgpu.NVOS54Parameters) (uintptr, error) {
```

Comment:
Nit: `AttachGpu` is a bit broad; I think `ctrlMemoryMulticastFabricAttachGPU` would be more specific and consistent with other `ctrl*` functions. Same for the Invoke helper function.

Reply:
Agree, thanks. Fixed in push.